.bazelci/presubmit.yml

```diff
@@ -32,3 +32,12 @@ tasks:
       - test
     build_targets:
       - //...
+  examples-nodejs:
+    name: Example - Node Helloworld
+    platform: ubuntu1804
+    working_directory: examples/node/coroutines-helloworld
+    include_json_profile:
+      - build
+      - test
+    build_targets:
+      - //coroutines-helloworld/...
```

Comment:
This is awesome.  Can you add building //express too?

Reply:
Will-do!